### PR TITLE
Hide Transcript from list of sub-objects on Generic Object

### PR DESCRIPTION
**Pull request by mkorcy**
This change will hide a transcript from the associated files list of a generic object when the primary object is a video and the transcript relationship is set to make the associated file list cleaner and easier to understand for users.

### CODE REVIEW

**Comment by rawOrlando** [comment]
Would you mind cleaning up the commits?

**Comment by mkorcy** [comment]
Theyr'e going to get squashed on  merge?

File: spec/presenters/hyrax/rcr_presenter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      end
+
+      it 'returns nil' do
+        expect(presenter.rcr_id).to be_nil
+      end
+    end
+
+    context 'when there is a file_set_presenter with mime_type "text/xml"' do
+      # rubocop:disable RSpec/VerifiedDoubles:
+      let(:xml_fsp) { double(id: '123', mime_type: 'text/xml') }
```
Comment (rawOrlando) [inline spec/presenters/hyrax/rcr_presenter_spec.rb:21]: did you forget `# rubocop:enable RSpec/VerifiedDoubles` ?

Comment (mkorcy) [reply spec/presenters/hyrax/rcr_presenter_spec.rb:21]: added

**Review by rawOrlando — APPROVED** [review]
🚀

**Pull request merged**